### PR TITLE
Upload artifacts to s3

### PR DESCRIPTION
**Pull request by wyrzyk**

### CODE REVIEW

File: scripts/upload-artifacts.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -ev
+export ARTIFACTS_BUCKET="jpt-travis-ci-pipeline"
```
Comment (dagguh) [inline scripts/upload-artifacts.sh:4]: "JPT Travis CI pipeline" has a teeny-tiny scent of Bitbucket Pipelines
```suggestion
export ARTIFACTS_BUCKET="jpt-travis-ci-artifacts"
```
Just a nit 😉

**Review by dagguh — APPROVED** [review]

**Review by dagguh — CHANGES_REQUESTED** [review]
It failed with:
```
./scripts/upload-artifacts.sh: line 5: artifacts: command not found
```

**Comment by wyrzyk** [comment]
AFAIK we can get artifacts upload out of the box in Circle CI.

**Comment by dagguh** [comment]
Correct. E.g.: https://circleci.com/gh/atlassian/aws-infrastructure/19#artifacts/containers/0

**Pull request closed without merging**